docs/set_esx_subnets.md

```diff
@@ -0,0 +1,25 @@
+# set_esx_subnets
+
+Sets the subnets that should be used to reach the ESXi hosts.
+```py
+def set_esx_subnets(esx_subnets=None, timeout=15)
+```
+
+## Keyword Arguments
+| Name        | Type | Description                                                                 | Choices | Default |
+|-------------|------|-----------------------------------------------------------------------------|---------|---------|
+| esx_subnets | str  | Preferred subnets used to reach the ESX hosts. The format should be the comma separated list without any spaces, for example, 192.168.2.10/24,10.255.0.2/16.  |         |         |
```

Comment:
`The format should be the comma separated list without any spaces, for example, 192.168.2.10/24,10.255.0.2/16. `

Why not just make this a list instead?

Reply:
The API request is expecting a string, are you suggesting taking a list and then concatenating the items into a string for the request? that will work too.

Reply:
Yeah, let’s automatically convert to a string based off of the provided list.